README.md

```diff
@@ -1,44 +1,111 @@
-* Remove USB stick and start system
-* Deloyment is performed, system shuts down afterwards
-* System is ready for use, by default DHCP is tried on
-  the 1st network device
-* Login via ``dragon`` and ``password``
-* There is a ``wireguard-client.conf`` that you can copy over to a client
-  to create a tunnel to access to the web interfaces and APIs of the system.
-  On the client, adjust the client IP in it (if you want to connect with multiple
-  clients, otherwise leave it alone), copy it to ``/etc/wireguard/wg0.conf`` and
-  ``wg-quick up wg0`` to start the tunnel.
-* Point your browser to the [homer dashboard](https://homer.services.in-a-box.cloud/) for
-  the admin web interfaces.
-* The installation is a minimalistic single-node installation similar to the
-  testbed installation. Consult the [testbed documentation](https://docs.osism.de/testbed/)
-  for an overview over the system. Default passwords can be found
-  [here](https://docs.osism.de/testbed/usage.html#webinterfaces).
+Cloud in a Box is a minimalistic installation of OSISM with only services which are needed to make it run.
+For more information you can consult the [Testbed](https://docs.osism.tech/testbed/index.html) documentation.
+If you want to have a full installation please have a look at the [OSISM](https://docs.osism.tech/) documentation.
+
+> **Warning:**
+> The secrets are stored in plain text and are not secure. Do not use for publicly
+> accessible systems.
+
+## Requirements
+
+* The first blockdevice is available as `/dev/sda` or `/dev/nvme0n1`.
+* USB stick with at least 2GB capacity.
+* CPU: 1 socket, 4 cores
+* RAM: 32 GB
+* Storage: 1 TB
+* Network: 1 network interfaces
+
+## Download the Cloud in a Box image
+
+> **Warning:** ⚡ When booting from this image, all data on the hard disks will be destroyed without confirmation.
+
+* <https://minio.services.osism.tech/node-image/ubuntu-autoinstall-cloud-in-a-box-1.iso> (with /dev/sda)
+* <https://minio.services.osism.tech/node-image/ubuntu-autoinstall-cloud-in-a-box-2.iso> (with /dev/nvme0n1)
+
+## Getting started
+
+Download the Cloud in a Box image from the link above.
+
+Use a tool to create a bootable USB stick with the Cloud in a Box image.
+
+Boot from the USB stick.
+
+The instalation will start and take a few minutes. After that the system will shutdown.
+
+Remove the USB stick and restart the system.
+
+The deployment will start. This takes some time and the system will shutdown when the deployment is finished.
+This takes roughly an hour, possibly longer depending on the hardware and internet connection.
+
+System is ready for use, by default DHCP is tried on the first network device.
+
+Login via ssh. Use the user **dragon** with the password **password**.
+
+```bash
+ssh dragon@IP_FROM_YOUR_SERVER
+```
+
+Copy the `/home/dragon/wireguard-client.conf` file to your device. This is necessary for using the web endpoints on your device.
+Rename the wireguard config file to something like `ciab.conf`.
```

Comment:
Maybe add:
If you want to connect to the cloud-in-a-box system from multiple clients, change the client IP in the config file to be different on each client.

Reply:
Sounds reasonable, I will add this to the documentation. Thanks for contributing!